code/modules/hydroponics/plant_genes.dm

```diff
@@ -99,6 +99,7 @@
 	name = "Nutriment"
 	var/reagent_id = "nutriment"
 	var/rate = 0.04
+	var/list/blacklisted_reagents = list("potassium", "cornoil", "diethylamine", "stable_plasma", "welding_fuel", "glycerol")
```

Comment:
Don't bananas have potassium? Try to blacklist them from separated reagents, not generally.